gapis/replay/replay_connection.go

```diff
@@ -108,6 +108,19 @@ func (e *backgroundConnection) HandlePayloadRequest(ctx context.Context, payload
 	return log.Errf(ctx, err, "Payload type is unexpected: %T", boxed)
 }
 
+// HandleProfilerReadyRequest implements gapir.ReplayResponseHandler interface.
+func (e *backgroundConnection) HandleProfilerReadyRequest(ctx context.Context, req *gapir.ProfilerReadyRequest, conn gapir.Connection) error {
+	ctx = status.Start(ctx, "Profile Request")
+	defer status.Finish(ctx)
+	waitId := req.GetId()
+	if waitId == 0 {
```

Comment:
Why is 0 a bad id?

Reply:
It's just an artifact of originally using notification IDs. I'll remove this check.